### DISCUSSION TITLE
3W data Outliers detection

### DISCUSSION THREAD

**Discussion by Shaima3j** [petrobras/3W#74] [Q&A]
Good day,

Are there any available outliers detection algorithms that use 3W as a database? can you share the algorithm or link to it.

Thank you.

**Comment by ricardoevvargas** [comment]
Hi, @Shaima3j.

That is a really interesting question. Thanks!

I don't remember any work that proposes a specific algorithm for outlier detection for 3W, but others may remember and comment here. 
Additionally, you can explore [this list of works that cite 3W](https://github.com/petrobras/3W/blob/main/LIST_OF_CITATIONS.md). If you find something, please register it here.
